include/log.h

```diff
@@ -99,6 +99,8 @@
 # endif
 #endif
 
+void pr_log_stacktrace(const char*moduleversion, const char*function);
```

Comment:
I like the general idea of this PR.  However, I'm wondering if a slightly different interface might work better.  Consider, for example, a signature of:

    void pr_log_stacktrace(int fd, const char *label, const char *file, int lineno);

That way, the stacktrace can be logged to an arbitrary file descriptor, not just via ProFTPD's logging routines.  Also, instead of providing the function name where an issue occurred, you provide the actual name of the file, and line number using the `__FILE__` and `__LINE__` preprocessor macros, _e.g._:

    /* Log the stacktrace */
    pr_log_stacktrace(log_fd, MOD_EXAMPLE_VERSION, __FILE__, __LINE__);

Or, even better:

    void pr_log_stacktrace(int fd, module *m, const char *file, int lineno);

Given a `module` pointer, the function could automatically create the "mod_example/version" string.